### PR TITLE
[IMP] cetmix_tower_server,cetmix_tower_server_notify_backend: Move notification features to new module

### PR DESCRIPTION
**Pull request by GabbasovDinar**
This commit moves all features that use the `web_notify` OCA module from the `cetmix_tower_server` module to the new module.

Task: 4000

### CODE REVIEW

File: cetmix_tower_server_notify_backend/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+{
+    "name": "Cetmix Tower Server Notify Backend",
+    "summary": "Backend notifications for Cetmix Tower",
+    "version": "14.0.1.0.0",
+    "category": "Productivity",
+    "website": "https://cetmix.com",
+    "author": "Cetmix",
+    "license": "AGPL-3",
+    "application": True,
```
Comment (ivs-cetmix) [inline cetmix_tower_server_notify_backend/__manifest__.py]: This is not an Application

File: cetmix_tower_server_notify_backend/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "installable": True,
+    "external_dependencies": {
```
Comment (ivs-cetmix) [inline cetmix_tower_server_notify_backend/__manifest__.py]: Please remove empty keys

File: cetmix_tower_server_notify_backend/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        "python": [],
+        "bin": [],
+    },
+    "depends": [
+        "cetmix_tower_server",
+        "web_notify",
+    ],
+    "data": [],
```
Comment (ivs-cetmix) [inline cetmix_tower_server_notify_backend/__manifest__.py]: Please remove empty keys

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
Please check the comments and also add `i18n` directory to the new module

**Review by ivs-cetmix — APPROVED** [review]
Code review LGTM

**Review by Aldeigja — APPROVED** [review]
Functional LGTM

**Comment by CetmixGitDrone** [comment]
On my way to merge this fine PR!
Prepared branch [14.0-dev-ocabot-merge-pr-122-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-122-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
@ivs-cetmix your merge command was aborted due to failed check(s), which you can inspect on [this commit of 14.0-dev-ocabot-merge-pr-122-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/0d56564030a34f47dbc6ae07e38ce1b62c28b061).

After fixing the problem, you can re-issue a merge command. Please refrain from merging manually as it will most probably make the target branch red.

**Comment by CetmixGitDrone** [comment]
Congratulations, PR rebased to [14.0-dev](https://github.com/cetmix/cetmix-tower/commits/14.0-dev).

**Comment by CetmixGitDrone** [comment]
This PR looks fantastic, let's merge it!
Prepared branch [14.0-dev-ocabot-merge-pr-122-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-122-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at d8d3a9ad4fc152ce6e6c9eac1ed320b852e53f21. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**